src/os/FileJournal.cc

```diff
@@ -866,6 +871,45 @@ int FileJournal::check_for_full(uint64_t seq, off64_t pos, off64_t size)
   return -ENOSPC;
 }
 
+int FileJournal::prepare_padding_entry(bufferlist& bl, off64_t& queue_pos)
```

Comment:
if they are not aligned, we would call align_bl the align them. Funcation align_bl would dp memcpy and  release some ptr. so this is not friendly to pcie ssd.
So if we use the pcie ssd, we should not use this patch.  user could set osd_journal_padding_ahead to true in pcie ssd.